config/config.go

```diff
@@ -222,6 +222,17 @@ func normalizeConfig(config *lint.Config) {
 			config.Rules[ruleName] = lint.RuleConfig{}
 		}
 	}
+	if config.EnableDefaultRules {
+		// Add to the configuration all default rules not yet present in it
+		for _, r := range defaultRules {
+			ruleName := r.Name()
+			_, alreadyInConf := config.Rules[ruleName]
+			if alreadyInConf {
+				continue
+			}
+			config.Rules[ruleName] = lint.RuleConfig{}
+		}
```

Comment:
This code block is identical to that a few lines above (except for the collection of rules we range over)
We could make an auxiliary method to factor out this common code and avoid the code duplication